src/index.js

```diff
@@ -1 +1,81 @@
 'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+const zlib = require('zlib');
+const { pipeline } = require('stream');
+
+const PORT = 5700;
+
+const server = http.Server();
+
+server.on('request', (req, res) => {
+  if(req.method === 'POST') {
+    const jsonPath = path.resolve('src', 'index.json');
+    const chunks = [];
+
+    if (!fs.existsSync(jsonPath)) {
+      res.readStatus = 500;
+      res.end()
+      return;
+    }
+    const jsonFile = fs.readFileSync(jsonPath, 'utf8');
+
+    if (!fs.existsSync(jsonFile)) {
```

Comment:
You are checking if the actual content of the file exists as a path, which is incorrect. This check is unnecessary and should be removed. You should receive data from form, pls delete json file